### DISCUSSION TITLE
Title h1 - h6

### DISCUSSION THREAD

**Discussion by krstivoja** [GrapesJS/grapesjs#4180] [Ideas]
I can see that everything is text based but I can not find adding h1, h2, h3 - h6. That is crucial for SEO. 

Any chance to add that?

**Comment by artf** [comment]
You can create your own component. Example: https://github.com/olivmonnier/grapesjs-plugin-header

**Reply by krstivoja** [reply]
Yes but can it be included in the core as that should be basics?
Thanks for the great work.
